### ISSUE TITLE
Option to exclude mobile devices

### ISSUE THREAD

**Issue by bombcheck** [loktar00/JQuery-Snowfall#50]
It would be great to have an option to exclude mobile devices (would be nicer to their cpu and batteries).
If anyone is interested: Check out this commit on my fork:
https://github.com/bombcheck/JQuery-Snowfall/commit/7aba203152ceeddedc67e0902f4dc6eb96dcacc1

**Comment by loktar00** [comment]
I agree this is a nice option, however I wonder if this is something the plugin should actually handle, or just the dev when instantiating it. It's trivial to wrap with a condition

```
if (!mobile) {
   $('whatever').snowFall();
}```

**Comment by bombcheck** [comment]
Sure. But on my opinion having a nice option to use is more straight forward.

**Comment by DanielRuf** [comment]
Useragent sniffing, not very reliable and not recommended.

We can use feature detection, battery API and so on. Because iPad and other devices also have a bad JS performance.
